tests/responses/__init__.py

```diff
@@ -33,10 +33,14 @@ def fake_response_from_file(file_name, url=None):
 
     file_content = open(file_path, 'r').read()
 
-    response = Response(
+    response = response_type(
         url=url,
         request=request,
-        body=file_content
+        body=file_content,
     )
-    response.encoding = 'utf-8'
+    try:
```

Comment:
If you set the default `response_type` to `TextResponse` and include `encoding='utf-8` in the instantiation, we can get rid of this part. (I think we can assume that all responses we want to mock this way is going to be a `TextResponse` as `HttpResponse` and `XMLResponse` inhertits from `TextReponse`)